### PR TITLE
The test was separated into differents describes

### PR DESCRIPTION
**Pull request by dennvm09**

### CODE REVIEW

File: test/buy-tshirt.spec.ts [hunk 1]
```diff
@@ -6,23 +6,42 @@ import {
...
+        beforeAll(async () => {
+          const shoppingCartPage: ShoppingCartPage = new ShoppingCartPage();
+          await shoppingCartPage.goToCheckout();
+        });
+        describe('Fill in the buyer information.', () => {
+          beforeAll(async () => {
+            const checkoutPage: CheckoutPage = new CheckoutPage();
+            await checkoutPage.goToSummary('Dennys', 'Mosquera', '720016');
+          });
+          describe('View purchase summary', () => {
```
Comment (kliver98) [inline test/buy-tshirt.spec.ts]: I think should be better `Go to checkout complete`

File: src/page/sign-in-step.page.ts [hunk 2]
```diff
@@ -16,11 +16,11 @@ export class SignInPage {
   }
 
   public async logIn(userName: string, passwd: string): Promise<void> {
-    await browser.wait(ExpectedConditions.presenceOf(this.userEle), 10000, 'It is not possible to access.');
+    await browser.wait(ExpectedConditions.presenceOf(this.userEle), 50000, 'It is not possible to access.');
```
Comment (kliver98) [inline src/page/sign-in-step.page.ts]: Why the change?

**Review by kliver98 — COMMENTED** [review]
Add screenshots of local and headless execution, images like this:
https://user-images.githubusercontent.com/36433884/125375935-9106de00-e34f-11eb-977c-60d03cab8ec3.png

File: test/buy-tshirt.spec.ts [hunk 3]
```diff
@@ -6,23 +6,42 @@ import {
...
+    beforeAll(async () => {
+      const signInPage: SignInPage = new SignInPage();
+      await signInPage.logIn('standard_user', 'secret_sauce');
+    });
+    describe('Add t-shirt to shopping cart.', () => {
+      beforeAll(async () => {
+        const inventoryPage: InventoryPage = new InventoryPage();
+        await inventoryPage.goToShoppingCart();
+      });
+      describe('View shopping cart and checkout.', () => {
```
Comment (kliver98) [inline test/buy-tshirt.spec.ts]: I think should be better `Go to checkout`

File: src/page/sign-in-step.page.ts (hunk 2 above)
Comment (dennvm09) [reply src/page/sign-in-step.page.ts]: It was an unnecessary change, I was testing something and I forgot to remove it.

**Comment by dennvm09** [comment]
**Local execution:**
image

**Headless execution:**
image

**Review by kliver98 — APPROVED** [review]
